main.py

```diff
@@ -189,6 +177,10 @@ def main():
     while True:
         process_keyboard_events(actions_while_key_pressed, actions_on_key_down)
         game_window.erase_screen()
+        ego.kinematic.external_forces.append(
+            CommonConstants.ROCKET_MASS
+            * Vec2d(0, CommonConstants.GRAVITATIONAL_FORCE_EARTH)
+        )
```

Comment:
Maybe in a later version we could move this to a general "apply_environmental_forces()" function to include other objects and forces while keeping the main loop as short and easy as possible

Reply:
yes, let's keep that in mind